modules/recommended-scale-practices.adoc

```diff
@@ -18,4 +18,14 @@ Cloud providers might implement a quota for API services. Therefore, gradually s
 
 The controller might not be able to create the machines if the replicas in the machine sets are set to higher numbers all at one time. The number of requests the cloud platform, which {product-title} is deployed on top of, is able to handle impacts the process. The controller will start to query more while trying to create, check, and update the machines with the status. The cloud platform on which {product-title} is deployed has API request limits and excessive queries might lead to machine creation failures due to cloud platform limitations.
 
-Enable machine health checks when scaling to large node counts. In case of failures, the health checks monitor the condition and automatically repair unhealthy machines.
+Enable machine health checks when scaling to large node counts. In case of failures, 
+the health checks monitor the condition and automatically repair unhealthy machines.
+
+[NOTE]
+====
+When scaling large and dense clusters to lower node counts, it might take large 
+amounts of time as the process involves draining or evicting the objects running on 
+the nodes being terminated in parallel and the client might start to throttle the 
+requests leading to queuing due to timeouts if there are too many objects to evict.
```

Comment:
I think this sentence is a bit awkward and I'm not sure how to rewrite it; should "leading to queuing" be "causing queuing"? 

timeouts, if there